src/main/java/mate/academy/Main.java

```diff
@@ -39,17 +53,43 @@ public static void main(String[] args) {
         tomorrowMovieSession.setMovie(fastAndFurious);
         tomorrowMovieSession.setShowTime(LocalDateTime.now().plusDays(1L));
 
-        MovieSession yesterdayMovieSession = new MovieSession();
-        yesterdayMovieSession.setCinemaHall(firstCinemaHall);
-        yesterdayMovieSession.setMovie(fastAndFurious);
-        yesterdayMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
+        MovieSession nextMovieSession = new MovieSession();
+        nextMovieSession.setCinemaHall(firstCinemaHall);
+        nextMovieSession.setMovie(fastAndFurious);
+        nextMovieSession.setShowTime(LocalDateTime.now().minusDays(1L));
 
-        MovieSessionService movieSessionService = null;
+        MovieSessionService movieSessionService =
+                (MovieSessionService) INJECTOR.getInstance(MovieSessionService.class);
         movieSessionService.add(tomorrowMovieSession);
-        movieSessionService.add(yesterdayMovieSession);
+        movieSessionService.add(nextMovieSession);
 
-        System.out.println(movieSessionService.get(yesterdayMovieSession.getId()));
+        System.out.println(movieSessionService.get(nextMovieSession.getId()));
         System.out.println(movieSessionService.findAvailableSessions(
-                        fastAndFurious.getId(), LocalDate.now()));
+                fastAndFurious.getId(), LocalDate.now()));
+
+        UserService userService =
+                (UserService) INJECTOR.getInstance(UserService.class);
+
+        AuthenticationService authenticationService =
+                (AuthenticationService) INJECTOR.getInstance(
+                        AuthenticationService.class);
+        try {
+            authenticationService.register(EMAIL, PASSWORD);
+        } catch (RegistrationException registrationException) {
+            System.err.println("Registration failed. This email: " + EMAIL + " already exists.");
+        }
+
+        User bob = userService.findByEmail(EMAIL).get();
+        ShoppingCartService cartService =
+                (ShoppingCartService) INJECTOR.getInstance(ShoppingCartService.class);
+
+        cartService.addSession(tomorrowMovieSession, bob);
+        cartService.addSession(nextMovieSession, bob);
+        ShoppingCart cart = cartService.getByUser(bob);
+
+        OrderService orderService =
+                (OrderService) INJECTOR.getInstance(OrderService.class);
+        orderService.completeOrder(cart);
+        orderService.getOrdersHistory(bob);
```

Comment:
The result of `orderService.getOrdersHistory(bob)` is not used or printed. If you intend to display the order history, consider printing the result or processing it further.